<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarModule</class>
 <widget class="QWidget" name="CalendarModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendario Radicale</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbar_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Calendario:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="calendar_selector"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_button">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_event_button">
       <property name="text">
        <string>Nuevo Evento</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="daily_view_container" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>